.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @salemove/tm-messenger
```

Comment:
Personal opinion: should be owned by @salemove/tm-quantum because it is related to exports

Reply:
Interesting, it's also a dependency in `engagement-archiver` but it doesn't seem to be used. 

I remember working with this library and discussing with the lovely maintainer [a while ago](https://github.com/felixbuenemann/xlsxtream/pull/52). Seems like it's following me.

Reply:
I'm also OK with Quantum owning it.

Reply:
> it's also a dependency in engagement-archiver but it doesn't seem to be used

Then it should be removed.

Reply:
Oh, looks like that was me. 😮 https://github.com/salemove/engagement-archiver/commit/01ac2b7335b4277333ac1b0044dcb6a409b6a98c#diff-ad5f389504142f2d0fe513a3e753360e16b47158e20cc11126d97c10680abd11

I removed the export engagement functionality in the repo since it was moved to engagement-registry.

Reply:
https://github.com/salemove/engagement-archiver/pull/291